locked-asset/token-unstake/src/lib.rs

```diff
@@ -0,0 +1,84 @@
+#![no_std]
+
+elrond_wasm::imports!();
+elrond_wasm::derive_imports!();
+
+pub const DEFAULT_UNBOND_EPOCHS: u64 = 10;
+
+#[derive(ManagedVecItem, TopEncode, TopDecode, NestedEncode, NestedDecode, TypeAbi, Clone)]
+pub struct UnstakePair<M: ManagedTypeApi> {
+    pub unlock_epoch: u64,
+    pub token_payment: EsdtTokenPayment<M>,
```

Comment:
I think we should just use `TokenAmountPair` struct here, as we don't need to keep storing nonces of 0 for every entry.

Reply:
I've kept it as a Payment, to more easily pass around the data, without the need to decompose and then create new payments again when needed.